lisa/stats.py

```diff
@@ -1057,15 +1135,27 @@ def format_val(val):
         if groups_as_row:
             facet_rows, collapse_cols = collapse_cols, facet_rows
 
+        # plotly backend supports another grouping layer, before having the
+        # need for collapsing columns.
+        tick_group = None
+        if backend != 'matplotlib':
+            for col in collapse_cols:
```

Comment:
It is a bit different. It can groups per tick several colors see the example below.

I can try to improve a bit the column selection though, by picking the one with the highest nunique() ?

![horizontal_bar-groups_as_row-matplotlib](https://user-images.githubusercontent.com/3084220/113305762-452dad00-92fb-11eb-86dc-cb4ca699a0c1.png)
![horizontal_bar-groups_as_row](https://user-images.githubusercontent.com/3084220/113305770-46f77080-92fb-11eb-8c11-1660b2cb4032.png)